### ISSUE TITLE
Allow to link features to specification

### ISSUE THREAD

**Issue by SebastianZ** [LeaVerou/css3test#179]
Not only the whole specifcation should be linked but also each individual feature that is tested.

For example, the `box-sizing` property should have a link to https://drafts.csswg.org/css-ui-3/#box-sizing, maybe also to https://www.w3.org/TR/css-ui-3/#box-sizing. This link might be implemented as a text or image badge shown when hovering the entry.

Sebastian

**Comment by Zefling** [comment]
It's complicated without a big change in the database (tests.js).

**Comment by SebastianZ** [comment]
Actually, I almost have a patch ready to optionally show those links, but yes, there needs to be a big change on tests.js to make this work.

I can do that but if you think it's not worth doing that for some reason, just let me know.

Sebastian

**Comment by Zefling** [comment]
You can request a pull request and we will be able to see. There, you can not tell if it's interesting.

**Comment by SebastianZ** [comment]
I've created the pull request for allowing to add the links:

https://github.com/LeaVerou/css3test/pull/180

I will work on providing a patch to update the data to include the links, but that'll take a bit.

Sebastian

**Comment by SebastianZ** [comment]
While creating https://github.com/LeaVerou/css3test/pull/181 I realized that there are some features listed in the wrong specification, some specification links are missing or are broken, and other such issues.

I'll file other patches to fix those once this huge PR is merged.

Sebastian

**Comment by Zefling** [comment]
It's not impossible. I never read around all the specs at one time.

A little change in navigation:
![2019-08-20_14h04_27](https://user-images.githubusercontent.com/3032555/63345775-cbc68e00-c353-11e9-9abe-724d5df787ab.png)

**Comment by SebastianZ** [comment]
> It's not impossible. I never read around all the specs at one time.

I don't get you here. What exactly is not impossible?

> A little change in navigation:
> ![2019-08-20_14h04_27](https://user-images.githubusercontent.com/3032555/63345775-cbc68e00-c353-11e9-9abe-724d5df787ab.png)

At first I thought the same but visually tying it to the name creates a stronger association with the feature and also makes it consistent with the placement of the links for the whole spec.

Sebastian

**Comment by SebastianZ** [comment]
As the spec. links are displayed on the page for quite some time now, I close this issue now.

@Zefling If you still think the links should be shown on the opposite size of the feature names, please create a new issue for that.

Sebastian
